packages/fluentui/react-northstar/src/components/Popup/Popup.tsx

```diff
@@ -374,7 +375,7 @@ export const Popup: React.FC<PopupProps> &
     return (
       !e.currentTarget ||
       !popupContentRef.current ||
-      (!e.currentTarget.contains(e.relatedTarget) && !popupContentRef.current.contains(e.relatedTarget))
+      (!elementContains(e.currentTarget, e.relatedTarget) && !elementContains(popupContentRef.current, e.relatedTarget))
```

Comment:
Is there a clear statement when we should use `elementContains()` vs `.contains()`?

Reply:
We should now always use `elementContains` so that the virtual-parent thing works nicely. I feel like React should have provided this function now that `createPortal` is a thing.